.travis.yml

```diff
@@ -2,15 +2,14 @@ language: python
 python:
   - "2.7"
   - "3.4"
+  - "3.6
 install:
     - pip install -r requirements.txt
     - pip install braintree coveralls mock mysqlclient
 env:
```

Comment:
You have assigned 3 values to the same env variable. Choose one, chose the database you want to use